### PR TITLE
Yonal development

### PR DESCRIPTION
**Pull request by yhern185**
Issue #121 pull request.
Please let me know if everything checks out fine.

### CODE REVIEW

File: src/en_US.json [hunk 1]
```diff
@@ -59,7 +59,11 @@
   "STDSelection": "Select The STD",
   "STDAwareness": "STD Awareness",
   "learnSTDs": "Learn about STDs and how they affect you",
+<<<<<<< HEAD
```
Comment (edumorlom) [inline src/en_US.json:62]: This won't compile, you have to fix this.

**Review by edumorlom — CHANGES_REQUESTED** [review]

**Comment by edumorlom** [comment]
I see you added the .vs directory to the PR, please remove that too

**Comment by yhern185** [comment]
Hi Eduardo, 

I can see the change as been introduced already.

Did you merge it?

**Comment by edumorlom** [comment]
What do you mean? It hasn't been merged yet. It's not in develop yet.

**Pull request closed without merging**